mava/systems/jax/ippo/system.py

```diff
@@ -25,7 +25,8 @@
 
 
 class IPPOSystem(System):
-    def design(self) -> Tuple[DesignSpec, Any]:
+    @staticmethod
```

Comment:
You would need to also make the updates in the super class system.py as it has `def design(self)`

Reply:
Great catch :raised_hands: I updated the code. I changed this as it would make defining systems easier in the future.